ui/packages/catalog-ui-search/src/main/webapp/js/CQLUtils.js

```diff
@@ -259,7 +267,11 @@ function isGeoFilter(type) {
 
 function transformFilterToCQL(filter) {
   return this.sanitizeGeometryCql(
-    '(' + cql.write(cql.simplify(cql.read(cql.write(filter)))) + ')'
+    '(' +
+      cql
+        .write(cql.simplify(cql.read(cql.write(filter))))
+        .replace(' null', '') +
```

Comment:
When the IS EMPTY query is constructed, it is built as `("location" IS NULL null)`, but that's an invalid query, you need to remove ` null` so that you get `("location" IS NULL)` which is the actual query you want to send

Reply:
✏️ See if you can change `cql.write` to not serialize `null` as a string and instead use the empty string.

Reply:
Currently looking into this